makefile

```diff
@@ -5,7 +5,27 @@ IMPORTS =
 ROKU_TEST_ID = 1
 ROKU_TEST_WAIT_DURATION = 5
 
+MAJOR = $(shell grep major_version manifest | sed 's/major_version=//')
```

Comment:
We should use the MAJOR, MINOR, BUILD_NUM and BUILD_HASH variables that are set by Jenkins, because we want to use TAR's version as part of the path, not the reference app's version.